go.mod

```diff
@@ -39,114 +39,123 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/ppc64le-cloud/powervs-utils v0.0.0-20230306072409-bc42a581099f
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.57.0
-	github.com/prometheus/client_golang v1.14.0
-	github.com/prometheus/client_model v0.3.0
-	github.com/prometheus/common v0.37.0
-	github.com/spf13/cobra v1.6.1
+	github.com/prometheus/client_golang v1.16.0
+	github.com/prometheus/client_model v0.4.0
+	github.com/prometheus/common v0.44.0
+	github.com/spf13/cobra v1.7.0
 	github.com/tombuildsstuff/giovanni v0.18.0
 	github.com/vincent-petithory/dataurl v1.0.0
-	go.uber.org/zap v1.21.0
+	go.uber.org/zap v1.25.0
 	golang.org/x/crypto v0.14.0
 	golang.org/x/net v0.17.0
 	golang.org/x/sync v0.4.0
 	golang.org/x/time v0.3.0
 	gopkg.in/ini.v1 v1.67.0
-	gopkg.in/square/go-jose.v2 v2.5.1
+	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.28.3
-	k8s.io/apiextensions-apiserver v0.25.0
+	k8s.io/apiextensions-apiserver v0.28.3
 	k8s.io/apimachinery v0.28.3
-	k8s.io/apiserver v0.25.0
-	k8s.io/cli-runtime v0.25.0
+	k8s.io/apiserver v0.28.3
+	k8s.io/cli-runtime v0.28.3
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/component-base v0.25.0
-	k8s.io/kube-aggregator v0.24.0
-	k8s.io/kube-scheduler v0.23.1
-	k8s.io/kubectl v0.25.0
+	k8s.io/component-base v0.28.3
+	k8s.io/kube-aggregator v0.28.3
+	k8s.io/kube-scheduler v0.28.3
+	k8s.io/kubectl v0.28.3
 	k8s.io/pod-security-admission v0.23.5
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	kubevirt.io/api v0.58.0
-	kubevirt.io/containerized-data-importer-api v1.50.0
+	kubevirt.io/api v1.0.0
```

Comment:
@davidvossel this ok with you?

Reply:
Required for the bump, so hopefully yes ... :)

Reply:
yeah definitely okay, thanks